easybuild/easyconfigs/t/TensorFlow-Datasets/TensorFlow-Datasets-4.7.0-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'PythonBundle'
+
+name = 'TensorFlow-Datasets'
+version = '4.7.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://www.tensorflow.org/datasets'
+description = """
+TensorFlow Datasets is a collection of datasets ready to use, with TensorFlow or other Python ML frameworks, such as
+Jax. All datasets are exposed as tf.data.Datasets , enabling easy-to-use and high-performance input pipelines.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('Python', '3.9.5'),
+    ('tqdm', '4.61.2'),
+    ('TensorFlow', '2.6.0', '-CUDA-%(cudaver)s'),
+]
+
+sanity_pip_check = True
+use_pip = True
+
+exts_list = [
+    ('promise', '2.3', {
+        'checksums': ['dfd18337c523ba4b6a58801c164c1904a9d4d1b1747c7d5dbf45b693a49d93d0'],
+    }),
+    ('googleapis-common-protos', '1.56.4', {
+        'modulename': 'google',
+        'checksums': ['c25873c47279387cfdcbdafa36149887901d36202cb645a0e4f29686bf6e4417'],
+    }),
+    ('tensorflow-metadata', '1.10.0', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'source_urls': [(
+            'https://files.pythonhosted.org/packages/3a/86/'
+            + '2b3251bb560068f31817d9b678588098e28f396c1f6b88c57cf5d28670af/'
+            + 'tensorflow_metadata-1.10.0-py3-none-any.whl'
+            + '#sha256=e3ff528496105c0d73b2a402877525b1695635378fbe5c1b47ac7b3780816bb3'
+        )],
```

Comment:
`tensorflow-metadata` is hosted on PYPI, so easybuild should be able to find it without specifying the `source_urls`
```suggestion
```

Reply:
Well I'm ready to yield that it should. But, I haven't managed to get it to work for tf metadata without specifying the url.

I'll accept the suggestion and if the download fails for you as well then perhaps another issue should be opened.

Reply:
ah I see the issue. adding `source_tmpl`: `tensorflow_metadata-%(version)s-py3-none-any.whl` should fix the download issue (it uses an underscore instead of a hyphen in the filename)